bukuserver/views.py

```diff
@@ -425,28 +415,32 @@ class TagModelView(BaseModelView, ApplyFiltersMixin):
     def _create_ajax_loader(self, name, options):
         pass
 
-    def _name_formatter(self, _, model, name):
+    def _name_formatter(self, context, model, name):
```

Comment:
This parameter actually _has to_ be named `context` for non-lazy `gettext()` to work (…and the lazy version ignores `escape()` somehow :sweat_smile:)